recipes/libpcap/all/conanfile.py

```diff
@@ -115,6 +117,11 @@ def generate(self):
                 "--disable-dbus",
                 "--disable-rdma",
             ])
+            
+            snf = self.options.get_safe("with_snf")
+            if snf:
+                tc.configure_args.append(f"--with-snf={yes_no(snf)}")
+            
```

Comment:
```suggestion
                "--disable-rdma",
                "--with-snf={yes_no(self.options.with_snf)}",
            ])
```

Reply:
Missing an `f` from the f-string.

Reply:
Why also `--disable-rdma`?

Reply:
`--disable-rdma` is just an existing unchanged line.